### ISSUE TITLE
apt update failing on maven:3.9.6-amazoncorretto-21-debian-bookworm

### ISSUE THREAD

**Issue by FlorianB-DE** [carlossg/docker-maven#522]
Getting build error using buildx:
```
0.398 Get:1 https://apt.corretto.aws stable InRelease [10.7 kB]
0.451 Err:1 https://apt.corretto.aws stable InRelease
0.451   The following signatures were invalid: EXPKEYSIG A122542AB04F24E3 Amazon Services LLC (Amazon Corretto release) <corretto-team@amazon.com>
0.582 Get:2 http://deb.debian.org/debian bookworm InRelease [151 kB]
0.599 Get:3 http://deb.debian.org/debian bookworm-updates InRelease [55.4 kB]
0.606 Get:4 http://deb.debian.org/debian-security bookworm-security InRelease [48.0 kB]
0.668 Get:5 http://deb.debian.org/debian bookworm/main amd64 Packages [8789 kB]
0.795 Get:6 http://deb.debian.org/debian bookworm-updates/main amd64 Packages [8856 B]
0.807 Get:7 http://deb.debian.org/debian-security bookworm-security/main amd64 Packages [236 kB]
1.688 Reading package lists...
2.125 W: GPG error: https://apt.corretto.aws stable InRelease: The following signatures were invalid: EXPKEYSIG A122542AB04F24E3 Amazon Services LLC (Amazon Corretto release) <corretto-team@amazon.com>
2.125 E: The repository 'https://apt.corretto.aws stable InRelease' is not signed.
```

**Comment by FlorianB-DE** [comment]
`docker run -it maven:3.9.6-amazoncorretto-21-debian-bookworm apt update`

**Comment by carlossg** [comment]
this is fixed in the 3.9.9 images, in previous ones you would need to update the key with something like

```
curl -fL https://apt.corretto.aws/corretto.key | gpg --batch --import \
&& gpg --batch --export '6DC3636DAE534049C8B94623A122542AB04F24E3' > /usr/share/keyrings/corretto.gpg
```

See #520
